pkgs/build-support/dotnet/nuget-to-nix/nuget-to-nix.sh

```diff
@@ -37,7 +39,7 @@ for package in *; do
   for version in *; do
     id=$(xq -r .package.metadata.id "$version/$package".nuspec)
 
-    if [[ -e "$excluded_source/$id.$version".nupkg ]]; then
+    if grep -q "$id.$version.nupkg" "$excluded_list"; then
```

Comment:
```suggestion
    if grep -qxF "$id.$version.nupkg" "$excluded_list"; then
```
* -x: Matches whole line (without this the pattern `x.1.0.0.nupkg` matches the file `fox.1.0.0.nupkg`
* -F: Interprets the pattern as a fixed string instead of a regular expression. Otherwise the `.` in the pattern will match any character.